modules/sandboxed-containers-upgrade-kataconfig-cli.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+// *upgrading-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-upgrade-kataconfig-cli_{context}"]
+= Patching the KataConfig custom resource (CR) using the CLI
+
+You can patch the `KataConfig` CR using the CLI.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the {product-title} CLI, run the following command:
++
+[source,terminal]
+----
+$ oc patch kataconfig <kataconfig_name> --type merge --patch
+'{"spec":{"kataMonitorImage":"registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.3.0"}}'
+----
+<1> Specifies the name of your Kata configuration file, such as `example-kataconfig`.
```

Comment:
You have an orphaned callout here. I suspect you are referring to `<kataconfig_name>`.

Maybe just, "where `kataconfig_name` specifies the name of your Kata configuration file, such as `example-kataconfig`.